src/plugins/intel_gpu/src/graph/eltwise.cpp

```diff
@@ -211,9 +211,9 @@ std::vector<layout> eltwise_inst::calc_output_layouts(eltwise_node const& /*node
             // value for every input
             auto in_shape = input_pshape.get_shape();
             for (size_t i = 0; i < desc->stride[0].spatial.size(); i++) {
-                const size_t idx = in_shape.size() - 1 - i;
-                if (idx < 0)
+                if (in_shape.size() < (1 + i))
```

Comment:
nit: Your change duplicates the condition `(1+i)` into two places. what about `const int idx = in_shape.size() - 1 -i` instead?

Reply:
Although it is unlikely to happen, it is possible for the idx value to be negative.